src/network_analyzer.cpp

```diff
@@ -702,6 +702,7 @@ NetworkAnalyzer::NetworkAnalyzer(struct iio_context *ctx, Filter *filt,
 
 	});
 
+	connect(this,SIGNAL(sweepDone()), ui->xygraph ,SLOT(reset()));
```

Comment:
3 different styles for separating with `,` used in one line. `a,b`, `a, b` and `a ,b`. Just use one. (prefered `a, b`)

Reply:
This was sloppy. Fixed.